src/posix/termios.rs

```diff
@@ -183,6 +184,12 @@ pub(crate) fn set_flow_control(termios: &mut Termios, flow_control: FlowControl)
     };
 }
 
+pub(crate) fn set_timeout(termios: &mut Termios, timeout: Duration) {
+    let timeout = u128::min(timeout.as_millis() / 100, u8::MAX as u128) as u8;
+    termios.c_cc[libc::VMIN as usize] = 0;
+    termios.c_cc[libc::VTIME as usize] = timeout;
```

Comment:
The description (and the documentation of termios as well) mention setting `VTIME` to 0 as well. What's the reason for setting it to `timeout` here?

Reply:
I made this utils function to bridge how timeouts are set on termios (by setting VTIME to a value with 100ms steps) with the Duration type on rust.
If you check at https://github.com/serialport/serialport-rs/blob/896136584a181581b04ef90279a5bfbd2cf50392/src/posix/tty.rs#L181, this function is being called with a duration = 0, which means that VTIME is set to zero.